docs/KubeConformance-1.10.md

```diff
@@ -0,0 +1,1256 @@
+# Kubernetes Conformance Test Suite - v1.9
```

Comment:
s/1.9/1.10

Is just this header wrong or was this generated based on v1.9? Lots of links seem to point to v1.10 so I'm guessing it is just the header.

Reply:
Hardcoded in the header, https://github.com/kubernetes/kubernetes/pull/72168  should fix

Reply:
#72168 doesn't seem to fix it. I manually edited again.

Run like this:

```
git checkout master
cp -a test/e2e/conformance test/e2e/conformance_walk
# ^^ so we can check out other versions and reference this updated walker/generatior
```

```
VERSION=1.15 # iterate through 1.10-1.15, 1.9 doesn't work with new code
git checkout v${VERSION}.0
go run test/conformance_walk/walk.go --conformance \
  --version v${VERSION} \
  --url https://github.com/kubernetes/kubernetes/tree/v${VERSION}.0/ \
  test/ \
  > ~/k8s-conformance/docs/KubeConformance-${VERSION}.md
grep Suite ~/k8s-conformance/docs/KubeConformance-$VERSION.md
# This always showed 1.9 in the documentation at the top
```